metricflow/sql/sql_column_type.py

```diff
@@ -0,0 +1,5 @@
+import datetime
+from typing import Union
+
+
+SqlColumnType = Union[str, int, float, datetime.datetime, datetime.date, bool]
```

Comment:
We might consider putting this into the sql_bind_parameters file

Reply:
Tried this, but it causes a circular import.